runtime/apis/tasksapi/handler.go

```diff
@@ -131,8 +131,25 @@ func Handler(s *proto.Schema) common.HandlerFunc {
 				return httpjson.NewErrorResponse(ctx, common.NewHttpMethodNotAllowedError("only HTTP GET or POST accepted"), nil)
 			}
 		case 3:
-			// TODO: POST topics/{name}/tasks/next - Assigns next task to me (or returns my current task)
-			return httpjson.NewErrorResponse(ctx, common.NewNotFoundError("Not found"), nil)
+			switch pathParts[2] {
+			case "next":
+				if r.Method != http.MethodPost {
+					return httpjson.NewErrorResponse(ctx, common.NewHttpMethodNotAllowedError("only HTTP POST accepted"), nil)
+				}
+
+				task, err := tasks.NextTask(ctx, identityID)
+				if err != nil {
+					if errors.Is(err, tasks.ErrTaskNotFound) {
+						return common.NewJsonResponse(http.StatusNotFound, map[string]any{}, nil)
```

Comment:
```suggestion
						return httpjson.NewErrorResponse(ctx, common.NewNotFoundError("Not found"), nil)
```
This should maintain the error structure format; e,.g.:

```
{
    "code": "ERR_RECORD_NOT_FOUND",
    "message": "Not found"
}```